english/marshmallow.md

```diff
@@ -975,9 +975,12 @@ You're reincarnated as a green starfish but wish you were [blue instead.](starfi
 
 [Motivate yourself](Motivate/Motivate.md)
 
-You wake up as Harry Potter ... and are facing Voldemort in a duel? 
+You wake up as Harry Potter ... and are facing Voldemort in a duel?
 What should you [do](magicaladventure/decision_time/decision_time.md)
 
 My tea's gone cold I'm wondering [why] (stan/stan.md) I got out of bed at all...
 
 [Click] (vote-the-president/bernie/Bernie.md) here to vote for Bernie Sanders.
+
+Are you a [girl](gender/girl.md)
+or a [boy](gender/boy.md)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.